common-test-lib/build.gradle

```diff
@@ -15,13 +15,13 @@
  */
 
 dependencies {
+    compile project(':google-cloud-tools-plugin:google-cloud-core')
```

Comment:
It looks like `common-test-lib` depends on `core` now, and `core` tests depend on `common-test-lib` in their turn. I can't remember off the top of my head if this is handled well by Gradle, what do you think, could this eventually lead to circular issues when compiling?